### DISCUSSION TITLE
装备拆解问题

### DISCUSSION THREAD

**Discussion by y-BraveShine** [LmeSzinc/AzurLaneAutoScript#1874] [Q&A] [answered]
昨天运行ALAS的时候偶然间看到游戏处于装备仓库界面，多了很多白色new装备，然后ALAS分解了其中一部分。但是我在设置里并没有看到有关于开科技箱和分解装备的设置。所以想问一下什么情况会触发装备分解？

**Comment by LmeSzinc** [comment, marked as answer]
在进行 E 系科研（装备解析）的时候。它们的前置条件是分解 8/15 件装备，Alas 会分解白色装备完成前置任务，白色装备不足时开白色箱子再分解。

**Reply by y-BraveShine** [reply]
感谢解答
